### DISCUSSION TITLE
Read-only prompt keys?

### DISCUSSION THREAD

**Discussion by Manouchehri** [langfuse/langfuse#1692] [Ideas]
### Describe the feature or potential improvement

When using prompt management, it seems like you have to give the application an API key, that has access to _all_ of the `/api/public/*` endpoints.

Would it be possible to create a read-only key that can _just_ access prompt management?

**Comment by marcklingen** [comment]
Thanks for raising this. This generalizes well to more narrowly scoped api keys

**Comment by zoltan-fedor** [comment]
Yes, I think there is a more generalized need to be able to set which Langfuse user of the project (and what API key) is allowed to modify prompts and which ones aren't.
I am imagining an individual dedicated to work on the prompts, so you might want to remove the ability to modify the prompts from all other team members.
